src/Phinx/Db/Adapter/AdapterInterface.php

```diff
@@ -78,6 +78,21 @@
     public function getVersions();
 
     /**
+     * Set adapter configuration options.
+     *
+     * @param  array $options
+     * @return AdapterInterface
+     */
+    public function setOptions(array $options);
+
+    /**
+     * Get adapter options.
+     *
+     * @return array
+     */
+    public function getOptions();
```

Comment:
These were defined in `PdoAdapter` but not in `AdapterInterface`. Since they are critical to `TablePrefixAdapter`, I figured they should be added to the interface.